feat: adding translation image, static, and bypass templates (#543)
aziontech/docs: src/content/docs/pt-br/pages/guias/marketplace/templates/bypass-suite/bypass-suite.mdx

---
title: Como implantar o template Suite Bypass

> description: O template inclui configurações que adicionam uma camada de engine rules para ignorar todas as configurações de cache para uma rota específica de URL.

Reviewer: ```suggestion
description: O template inclui configurações que adicionam uma camada de regras para ignorar todas as configurações de cache para uma rota específica de URL.
```

meta_tags: templates, guides, Azion Marketplace
namespace: docs_guides_bypass_template
permalink: /documentacao/produtos/guias/suite-bypass/
---

import Badge from '~/components/Badge.astro';

<Badge variant="accent">
Preview
</Badge>

> **Suite Bypass** inclui configurações que adicionam uma camada de engine rules para ignorar todas as configurações de cache para uma rota específica de URL. Essas configurações utilizam uma regra padrão baseada em critérios e comportamentos aplicados por meio do Rules Engine para Edge Application.

Reviewer: ```suggestion
**Suite Bypass** inclui configurações que adicionam uma camada de regras para ignorar todas as diretivas de cache para uma rota específica de URL. Essas configurações utilizam uma regra padrão baseada em critérios e comportamentos aplicados por meio do Rules Engine para Edge Application.
```

A regra é composta por:

- **Criteria** (critério): if `${uri}` *starts with* `/url_route`.
- **Behavior** (comportamento): then [Set Origin](/pt-br/documentacao/produtos/edge-application/rules-engine/#set-origin) `Default Origin`, [Bypass Cache](/pt-br/documentacao/produtos/edge-application/rules-engine/#bypass-cache) e [Forward Cookies](/pt-br/documentacao/produtos/edge-application/rules-engine/#forward-cookies).

Você deve vincular sua origem ao template para começar a usar este conjunto de regras de cache para entrega. Após o deployment, o script cria uma nova aplicação edge que é executada na Plataforma de Edge Computing da Azion, com a qual você pode gerenciar e ajustar as configurações.

---

## Obtenha o template

Para usar este template, acesse o [Real-Time Manager (RTM)](https://manager.azion.com/) e selecione a opção **Start with a template** na página inicial.

Isso abrirá uma nova página onde você pode selecionar o card do **Suite Bypass**.

---

## Configure o template

> Uma vez que você está na página do template, clique na aba **Settings** para abrir o formulário de configuração e fornecer as informações para configurar sua aplicação.

Reviewer 2: ```suggestion
Na página do template, clique na aba **Settings** para abrir o formulário de configuração e fornecer as informações para configurar sua aplicação.
```

---

## Implante o template

Durante a implantação, você poderá acompanhar o processo através de uma janela mostrando os logs. Quando estiver concluido, aparecerá uma nova página confirmando que a edge application foi criada com sucesso.

Esta página mostra as seguintes seções:

- A seção **Access your Edge Application**, que inclui o link para visitar e explorar sua aplicação.
- O **Build Summary** contém o nome e a função da aplicação.
- A aba **Deployment details** pode ser aberta para acessar os logs relacionados à implantação.
- **What do you want to do next?** fornece recomendações sobre opções avançadas para aprimorar sua edge application: **Customize Domain**, **Manage Edge Application**, **View Edge Application Metrics**, **View Edge Application Logs** e o botão **Back to Home**.

:::note
O link para sua edge application permite que você veja como ela fica no navegador. No entanto, leva um certo tempo para propagar e configurá-la nas edge locations da Azion. Pode ser necessário aguardar alguns minutos para que a URL seja ativada e para que a página da aplicação seja efetivamente exibida no navegador.
:::

> ### Principais configurações

Reviewer: ```suggestion
### Configurações principais
```

3. Na seção de **BUILD**, selecione **Edge Application**.
4. Você será redirecionado para a página de **Edge Application**.
- Ela lista todas as edge applications que você criou.
5. Encontre a aplicação relacionada ao seu template e selecione-a.
6. Na aba **Main Settings**, encontre a seção **Edge Application Modules** e ative o switch dos módulos.

:::caution[Atenção]
Se estes módulos estiverem ativados, a execução deste template pode gerar custos relacionados ao uso. Consulte a [página de preços](https://www.azion.com/pt-br/precos/) para obter mais informações.
:::

---

## Gerencie o template

Considerando que essa configuração inicial pode não ser ideal para sua aplicação, todas as configurações podem ser personalizadas sempre que você precisar usando o RTM.

Para gerenciar e editar as configurações da sua aplicação, siga estas etapas:

1. [Acesse o RTM](https://manager.azion.com/).
2. No canto superior esquerdo, selecione as três linhas horizontais para abrir o **Products menu**.
3. Na seção de **BUILD**, selecione **Edge Application**.
4. Você será redirecionado para a página de **Edge Application**.
- Ela lista todas as edge applications que você criou.

> 5. Encontre a aplicação relacionada ao **Fauna DB Boilerplate* e selecione-a.

Reviewer 2: ```suggestion
5. Encontre a aplicação relacionada ao ** Suite Bypass** e selecione-a.
```

- A lista é organizada em ordem alfabética. Você também pode usar a **barra de busca** localizada no canto superior esquerdo da lista; atualmente, ela é filtrada apenas pelo **Application Name**, ou nome da edge application.

> Depois de selecionar a aplicação em que você trabalhará, você será direcionado para uma página que contém todas as configurações que você pode configurar.

Reviewer: ```suggestion
Depois de selecionar a aplicação desejada, você será direcionado para uma página que contém todas as configurações que você pode configurar.
```
